src/main/java/com/microsoft/jfr/FlightRecorderConnection.java

```diff
@@ -143,6 +136,30 @@ public long startRecording(RecordingOptions recordingOptions, RecordingConfigura
         }
     }
 
+    private void setConfiguration(RecordingConfiguration recordingConfiguration, long id) throws OpenDataException, InstanceNotFoundException, MBeanException, ReflectionException, IOException {
+        if (recordingConfiguration == null || recordingConfiguration.getConfiguration() == null) {
+            return;
```

Comment:
Should we log this no-op?

Reply:
I don't see the use of logger in Microsoft JFR streaming. @dsgrieve  and @johnoliver, what do you think about adding a logger here?

Reply:
If logging is added, then it needs to be done in a portable way, such as apache commons logging api, and the actual logger would be injected. 

But, again, if this is meant to be a utility that is used as part of a build process as opposed to an integral part of the library, then any logger would do.

Reply:
@dsgrieve - Your architectural decision on whether to add logging to this lib :-).

Reply:
Done